### PR TITLE
Releases/v1 updates 

### PR DESCRIPTION
**Pull request by kanika1894**
1. Added support for using environment variables as GITHUB_ENV
2. Added workflow files for build, stale issues

(To release 1.0.4 version)

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
@kanika1894 shouldn't we ignore node_modules folder?

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,7 @@
 *.userosscache
 *.sln.docstates
 
-lib
+#lib
```
Comment (bishal-pdMSFT) [inline .gitignore:12]: remove this line

File: .gitignore [hunk 2]
```diff
@@ -263,7 +263,7 @@ FakesAssemblies/
 
 # Node.js Tools for Visual Studio
 .ntvs_analysis.dat
-node_modules/
+#node_modules/
```
Comment (bishal-pdMSFT) [inline .gitignore:266]: why is this commented?

Comment (kanika1894) [reply .gitignore]: updated

**Review by bishal-pdMSFT — APPROVED** [review]

**Pull request merged**